.gitignore

```
@@ -45,3 +45,7 @@ nbactions.xml
 .DS_Store
 .attach_pid*
 
+# Bitronix transaction logs
+part1.btm
+part2.btm
```

There may be a `part3` from time to time? Better safe than sorry:

```suggestion
*.btm
```